pkg/pillar/types/zedagenttypes.go

```diff
@@ -219,13 +219,15 @@ const (
 )
 
 type DatastoreConfig struct {
-	UUID     uuid.UUID
-	DsType   string
-	Fqdn     string
-	ApiKey   string
-	Password string
-	Dpath    string // depending on DsType, it could be bucket or path
-	Region   string
+	UUID        uuid.UUID
+	DsType      string
+	ApiKey      string
+	Password    string
+	Fqdn        string
+	Dpath       string // depending on DsType, it could be bucket or path
+	Region      string
+	IsCipher    bool
```

Comment:
Please add comment stating that when IsCipher is set, the the CipherBlock contains the ApiKey and Password and the above ApiKey and Password strings are empty/ignored.

Reply:
will do.